### ISSUE TITLE
Calculating training metrics

### ISSUE THREAD

**Issue by ggerogiokas** [unit8co/darts#448]
Hi,

Thanks for the great work on this library.  

I was playing around with darts I can see lots of great functionality on backtests, historical_forecasts,
but it's not clear what the recommended way to get training metrics.

Best,
George

**Comment by hrzn** [comment]
Hello,
Could you clarify what you mean by training metrics?

**Comment by ggerogiokas** [comment]
Hi,

Sure, I mean getting a MAE, MSE or any other metric on the training set.  The closest thing I could find was the residual function.  Also would be interested to get the loss during the training.  I saw you can get them for the NNs with tensorboard but didn't notice how to do it for other models.

Thanks,
George
